addons/project_purchase/static/src/product_catalog/kanban_record.js

```diff
@@ -0,0 +1,16 @@
+/** @odoo-module **/
+import { patch } from "@web/core/utils/patch";
+
+import { ProductCatalogKanbanRecord } from "@product/static/src/product_catalog/kanban_record";
+
+patch(ProductCatalogKanbanRecord.prototype, {
+    _updateQuantityAndGetPrice() {
+        return rpc("/product/catalog/update_order_line_info", {
+            order_id: this.env.orderId,
+            product_id: this.env.productId,
+            quantity: this.productCatalogData.quantity,
+            res_model: this.env.orderResModel,
+            project_id: this.props.record.context.project_id,
+        });
```

Comment:
I think it would be better to add a hook in sale module and override that hook in that module as we got in 17.2+ 🙂 

<https://github.com/odoo/odoo/blob/3f39aa7504d95ea93f64e62ebdae6c980116dcd4/addons/product/static/src/product_catalog/kanban_record.js#L71-L78>

Reply:
Also, this file is not imported in the assets inside the manifest. 😕